locale/es.po

```diff
@@ -1345,14 +1339,12 @@ msgid "File exists"
 msgstr ""
 
 #: py/moduerrno.c:148
-#, fuzzy
 msgid "Unsupported operation"
-msgstr "El pin %d no soporta PWM"
+msgstr "Operacion no soportada"
 
 #: py/moduerrno.c:149
-#, fuzzy
 msgid "Invalid argument"
-msgstr "argumentos inválidos"
+msgstr "argumento inválido"
```

Comment:
Maycula la primera letra, segun como se ha estado manejando

Reply:
Fixed in c16ef42